### PR TITLE
updating gpt3.5 api as per latest version

### PR DESCRIPTION
**Pull request by mrunal-create**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -19,34 +19,63 @@ class ResumeData(BaseModel):
...
-                {"role": "system", "content": "You are a coding and system design expert."},
-                {"role": "user", "content": prompt}
-            ]
-        )
-        formatted_response = response['choices'][0]['message']['content'].strip()
-        return formatted_response
+        client = OpenAI(
+    # This is the default and can be omitted
+    api_key= OPENAI_API_KEY,) 
+        chat_completion = client.chat.completions.create(
```
Comment (deveshcode) [inline main.py:32]: Tab's  are not aligned .. api key and client check invalid tabbing

File: main.py
Comment (deveshcode) [inline main.py:1]: Remove the commented code its of no use.

**Review by deveshcode — COMMENTED** [review]
Please take a look at my comments. And reraise.

File: main.py
Comment (mrunal-create) [reply main.py:1]: doneeeeeeeeeee

File: main.py (hunk 1 above)
Comment (mrunal-create) [reply main.py:32]: doneeeeeeee

**Review by mrunal-create — COMMENTED** [review]
doneeeeee

**Pull request merged**